src/Host/Controllers/Catalog/BrandsController.cs

```diff
@@ -45,7 +45,7 @@ public Task<string> GenerateRandomAsync(GenerateRandomBrandRequest request)
     }
 
     [HttpDelete("delete-random")]
-    [ApiConventionMethod(typeof(FSHApiConventions), nameof(FSHApiConventions.Delete))]
+    [ApiConventionMethod(typeof(FSHApiConventions), nameof(FSHApiConventions.Search))]
```

Comment:
Search?

Reply:
The problem is that you can't use a method like that that has multiple overloads... and search is one that doesn't have multiple overloads...

Reply:
At this point it doesn't really matter as they all have the same attributes... but eventually we will probably want to clean this out and put the right responsecodes on the right kind of methods...